utils.py

```diff
@@ -75,7 +75,7 @@ def get_new_recovered(self):
             self.recovered.append(recovered_person)
 
 
-def simulate(model, df, timestamps):
+def simulate(model, df, timestamps, vaccinated, d):
```

Comment:
Can we have a more explanatory variable name instead of `d`?
Something like `vaccination_day`?